src/Numerics/DeterminantOperators.h

```diff
@@ -283,5 +304,35 @@ inline void InverseUpdateByColumn(MatA& Minv,
   //for(int k=0; k<nrows; k++) Minv(k,colchanged) *= ratio_inv;
 }
 
+template<typename MatA, typename VecT>
+inline void mw_InverseUpdateByColumn(const int nw,
+                                     RefVector<MatA>& Minv_list,
+                                     RefVector<VecT>& newcol_list,
+                                     RefVector<VecT>& rvec_list,
+                                     RefVector<VecT>& rvecinv_list,
+                                     int colchanged,
+                                     RefVector<typename MatA::value_type> c_ratio_list)
+{
+#pragma omp parallel for
+  for (size_t iw = 0; iw < nw; iw++)
+    det_col_update(Minv_list[iw].get().data(), newcol_list[iw].get().data(), Minv_list[iw].get().rows(), colchanged,
+                   c_ratio_list[iw].get(), rvec_list[iw].get().data(), rvecinv_list[iw].get().data());
+}
+
+
+template<typename MatA, typename VecT>
+inline void mw_InverseUpdateByColumn(const int nw,
+                                     RefVector<MatA>& Minv_list,
+                                     RefVector<VecT>& newcol_list,
+                                     RefVector<VecT>& rvec_list,
+                                     RefVector<VecT>& rvecinv_list,
+                                     int colchanged,
+                                     std::vector<typename MatA::value_type> ratioGradReflistIdim)
```

Comment:
Remove the above mw_InverseUpdateByColumn and keep only this one. Also change template typenames to a value type shared by both matrices and vectors.

Reply:
RefVector -> const RefVector

Reply:
Done